### PR TITLE
Add and Update several password change urls

### PR DESCRIPTION
**Pull request by joshka**
### Overall Checklist
- [x] I agree to the project's [Developer Certificate of Origin](https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md)
- [x] The top-level JSON objects are sorted alphabetically
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update

#### for change-password-URLs.json
- [x] There is no Well-Known URL for Changing Passwords (`https://example.com/.well-known/change-password`)
- [x] The URL either makes the experience better or no worse than being directed to just the domain in a non-logged-in state

### CODE REVIEW

**Comment by simap-ag** [comment]
Can you remove  commit [03e1675](https://github.com/apple/password-manager-resources/pull/665/commits/03e167596d9e0a1984cab37e9cfab27fd64cc747), as I have it in another pull request already.

**Comment by joshka** [comment]
> Can you remove commit [03e1675](https://github.com/apple/password-manager-resources/pull/665/commits/03e167596d9e0a1984cab37e9cfab27fd64cc747), as I have it in another pull request already.

Done.

File: quirks/change-password-URLs.json [hunk 1]
```diff
@@ -41,20 +47,23 @@
     "clien.net": "https://www.clien.net/service/mypage/myInfoComfrim",
     "cloudflare.com": "https://dash.cloudflare.com/profile/authentication",
     "codepen.io": "https://codepen.io/settings/account",
+    "columbia.com": "https://www.columbia.com/profile",
     "consumidor.gov.br": "https://www.consumidor.gov.br/pages/usuario/editar",
     "coupang.com": "https://login.coupang.com/login/userModify.pang",
+    "crackle.com": "https://www.crackle.com/profile",
```
Comment (rmondello) [inline quirks/change-password-URLs.json:53]: Can we remove `www.`?

File: quirks/change-password-URLs.json [hunk 2]
```diff
@@ -41,20 +47,23 @@
     "clien.net": "https://www.clien.net/service/mypage/myInfoComfrim",
     "cloudflare.com": "https://dash.cloudflare.com/profile/authentication",
     "codepen.io": "https://codepen.io/settings/account",
+    "columbia.com": "https://www.columbia.com/profile",
```
Comment (rmondello) [inline quirks/change-password-URLs.json:50]: Can we remove `www.`?

File: quirks/change-password-URLs.json [hunk 3]
```diff
@@ -29,9 +32,12 @@
     "benefitslogin.discoverybenefits.com": "https://benefitslogin.discoverybenefits.com/Profile/UpdatePassword.aspx",
     "berlet.de": "https://www.berlet.de/mein-konto.htm#my-account--edit-pass",
     "birkenstock.com": "https://www.birkenstock.com/profile",
+    "blend.io": "https://blend.io/settings",
     "blockchain.com": "https://login.blockchain.com/en/#/security-center/advanced",
     "blutdruck-shop.de": "https://www.blutdruck-shop.de/mein-passwort/",
     "browserstack.com": "https://www.browserstack.com/accounts/profile",
+    "bugzilla.kernel.org": "https://bugzilla.kernel.org/userprefs.cgi?tab=account",
+    "cakeresume.com": "https://www.cakeresume.com/settings/account?ref=navs_settings",
```
Comment (rmondello) [inline quirks/change-password-URLs.json:40]: Can we remove `www.`?

File: quirks/change-password-URLs.json [hunk 4]
```diff
@@ -11,10 +11,13 @@
     "aerlingus.com": "https://www.aerlingus.com/html/user-profile.html",
     "aesop.com": "https://www.aesop.com/my-account",
     "airbnb.com": "https://www.airbnb.com/account-settings/login-and-security",
+    "airnewzealand.com": "https://www.airnewzealand.com/membership/profile/security/password",
+    "alaskaair.com": "https://www.alaskaair.com/www2/ssl/myalaskaair/myalaskaair.aspx?view=myinformation&tab=email",
```
Comment (rmondello) [inline quirks/change-password-URLs.json:15]: Can we remove `www.`?

File: quirks/change-password-URLs.json [hunk 5]
```diff
@@ -11,10 +11,13 @@
     "aerlingus.com": "https://www.aerlingus.com/html/user-profile.html",
     "aesop.com": "https://www.aesop.com/my-account",
     "airbnb.com": "https://www.airbnb.com/account-settings/login-and-security",
+    "airnewzealand.com": "https://www.airnewzealand.com/membership/profile/security/password",
```
Comment (rmondello) [inline quirks/change-password-URLs.json:14]: Can we remove `www.`?

**Review by rmondello — CHANGES_REQUESTED** [review]
Looks great to me. Awesome contribution! I left a few comments about `www.`. After addressing some of those, I’d be thrilled to merge this.

**Comment by joshka** [comment]
Removing www seems generally wrong to me on these. The sites in question have chosen to use the prefix as their canonical url (all urls are copied from a live session using the site). The non-www version might work in some cases, but is not the url that the sites advertise themselves. I get the arguments both ways on no-www and yes-www sides of the issue (I lean no-www, but not enough to try to argue for it), but the decision is the target website owner’s not the browser / tool maker’s.

This said - it’s entirely possible there’s something technical I’m missing that’s not stated in why you want the www dropped - help me understand if I have?

**Comment by rmondello** [comment]
@joshka If these are, in fact, the canonical URLs, then this is great!

**Review by rmondello — APPROVED** [review]

**Comment by rmondello** [comment]
Thanks for the contribution, @joshka!

**Pull request merged**